bdPostgres.js

```diff
@@ -0,0 +1,8 @@
+const {Pool} = require('pg');
+const pool = new Pool({
+
+
+})
```

Comment:
why all these empty lines?

```suggestion
const pool = new Pool({})
```